### ISSUE TITLE
Can't run "sbt webStage"

### ISSUE THREAD

**Issue by ignasi35** [lagom/lagom.github.io#229]
Trying to run `sbt webStage` locally fails with :

```
[error] internal/fs/utils.js:701
[error]     throw new ERR_INVALID_ARG_TYPE(
[error]     ^
[error]
[error] TypeError [ERR_INVALID_ARG_TYPE]: The "data" argument must be of type string or an instance of Buffer, TypedArray, or DataView. Received an instance of Object
[error]     at Object.writeFileSync (fs.js:1409:5)
```

(full stack trace below)

### Environment:

```
$ node --version
v14.11.0

$ java -version
openjdk version "1.8.0_265"
OpenJDK Runtime Environment (AdoptOpenJDK)(build 1.8.0_265-b01)
OpenJDK 64-Bit Server VM (AdoptOpenJDK)(build 25.265-b01, mixed mode)

$ alias sbt
sbt=sbtx
```

I also checked in `vegemite` (the build machine publishing the site nightly), where no error is reported yet:

```
$ node --version
v11.1.0

$ java -version
java version "1.8.0_171"
Java(TM) SE Runtime Environment (build 1.8.0_171-b11)
Java HotSpot(TM) 64-Bit Server VM (build 25.171-b11, mixed mode)
```

<details>
<summary>Full output</summary>

```
sbt webStage
[info] Loading settings for project global-plugins from dependencyGraph.sbt ...
[info] Loading global plugins from /Users/ignasi/.sbt/1.0/plugins
[info] Loading settings for project template-build from plugins.sbt ...
[info] Loading project definition from /Users/ignasi/git/github/lagom/lagom.github.io/template/project
[info] Loading settings for project lagom-docs from build.sbt ...
[info] Set current project to lagom-docs (in build file:/Users/ignasi/git/github/lagom/lagom.github.io/template/)
[info] Updating ...
[info] Done updating.
[warn] There may be incompatibilities among your library dependencies; run 'evicted' to see detailed eviction warnings.
[info] Compiling 36 Scala sources to /Users/ignasi/git/github/lagom/lagom.github.io/template/target/scala-2.12/classes ...
[info] Stylus compiling on 1 source(s)
[error] (node:60762) Warning: Accessing non-existent property 'lineno' of module exports inside circular dependency
[error] (Use `node --trace-warnings ...` to show where the warning was created)
[error] (node:60762) Warning: Accessing non-existent property 'column' of module exports inside circular dependency
[error] (node:60762) Warning: Accessing non-existent property 'filename' of module exports inside circular dependency
[error] (node:60762) Warning: Accessing non-existent property 'lineno' of module exports inside circular dependency
[error] (node:60762) Warning: Accessing non-existent property 'column' of module exports inside circular dependency
[error] (node:60762) Warning: Accessing non-existent property 'filename' of module exports inside circular dependency
[info] Optimizing 1 JavaScript(s) with Uglify
[error] WARN: Collapsing variable iOS [main.js:9,11]
[error] internal/fs/utils.js:701
[error]     throw new ERR_INVALID_ARG_TYPE(
[error]     ^
[error]
[error] TypeError [ERR_INVALID_ARG_TYPE]: The "data" argument must be of type string or an instance of Buffer, TypedArray, or DataView. Received an instance of Object
[error]     at Object.writeFileSync (fs.js:1409:5)
[error]     at done (/Users/ignasi/git/github/lagom/lagom.github.io/template/project/target/node-modules/webjars/uglify-js/bin/uglifyjs:516:20)
[error]     at cb (/Users/ignasi/git/github/lagom/lagom.github.io/template/project/target/node-modules/webjars/uglify-js/bin/uglifyjs:324:39)
[error]     at /Users/ignasi/git/github/lagom/lagom.github.io/template/project/target/node-modules/webjars/uglify-js/bin/uglifyjs:391:9
[error]     at FSReqCallback.readFileAfterClose [as oncomplete] (internal/fs/read_file_context.js:63:3) {
[error]   code: 'ERR_INVALID_ARG_TYPE'
[error] }
[info] Done compiling.
[info] Packaging /Users/ignasi/git/github/lagom/lagom.github.io/template/target/scala-2.12/lagom-docs_2.12-0.1.0-SNAPSHOT.jar ...
[info] Done packaging.
[info] Running com.lightbend.lagom.docs.DocumentationGenerator /Users/ignasi/git/github/lagom/lagom.github.io/template/target/web/generated-html /Users/ignasi/git/github/lagom/lagom.github.io/template/src/docs /Users/ignasi/git/github/lagom/lagom.github.io/template/src/main/markdown /Users/ignasi/git/github/lagom/lagom.github.io/template/src/blog 915635b767f34655325b4ad249b57fa06ba4b6fe
[error] com.typesafe.sbt.jse.SbtJsTask$JsTaskFailure: WARN: Collapsing variable iOS [main.js:9,11]
[error] internal/fs/utils.js:701
[error]     throw new ERR_INVALID_ARG_TYPE(
[error]     ^
[error]
[error] TypeError [ERR_INVALID_ARG_TYPE]: The "data" argument must be of type string or an instance of Buffer, TypedArray, or DataView. Received an instance of Object
[error]     at Object.writeFileSync (fs.js:1409:5)
[error]     at done (/Users/ignasi/git/github/lagom/lagom.github.io/template/project/target/node-modules/webjars/uglify-js/bin/uglifyjs:516:20)
[error]     at cb (/Users/ignasi/git/github/lagom/lagom.github.io/template/project/target/node-modules/webjars/uglify-js/bin/uglifyjs:324:39)
[error]     at /Users/ignasi/git/github/lagom/lagom.github.io/template/project/target/node-modules/webjars/uglify-js/bin/uglifyjs:391:9
[error]     at FSReqCallback.readFileAfterClose [as oncomplete] (internal/fs/read_file_context.js:63:3) {
[error]   code: 'ERR_INVALID_ARG_TYPE'
[error] }
[error]
[error] 	at com.typesafe.sbt.jse.SbtJsTask$.$anonfun$executeJsOnEngine$1(SbtJsTask.scala:218)
[error] 	at scala.util.Success.$anonfun$map$1(Try.scala:251)
[error] 	at scala.util.Success.map(Try.scala:209)
[error] 	at scala.concurrent.Future.$anonfun$map$1(Future.scala:288)
[error] 	at scala.concurrent.impl.Promise.liftedTree1$1(Promise.scala:29)
[error] 	at scala.concurrent.impl.Promise.$anonfun$transform$1(Promise.scala:29)
[error] 	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:60)
[error] 	at java.util.concurrent.ForkJoinTask$RunnableExecuteAction.exec(ForkJoinTask.java:1402)
[error] 	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:289)
[error] 	at java.util.concurrent.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1056)
[error] 	at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1692)
[error] 	at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:175)
[error] (webPipeline) com.typesafe.sbt.jse.SbtJsTask$JsTaskFailure: WARN: Collapsing variable iOS [main.js:9,11]
[error] internal/fs/utils.js:701
[error]     throw new ERR_INVALID_ARG_TYPE(
[error]     ^
[error]
[error] TypeError [ERR_INVALID_ARG_TYPE]: The "data" argument must be of type string or an instance of Buffer, TypedArray, or DataView. Received an instance of Object
[error]     at Object.writeFileSync (fs.js:1409:5)
[error]     at done (/Users/ignasi/git/github/lagom/lagom.github.io/template/project/target/node-modules/webjars/uglify-js/bin/uglifyjs:516:20)
[error]     at cb (/Users/ignasi/git/github/lagom/lagom.github.io/template/project/target/node-modules/webjars/uglify-js/bin/uglifyjs:324:39)
[error]     at /Users/ignasi/git/github/lagom/lagom.github.io/template/project/target/node-modules/webjars/uglify-js/bin/uglifyjs:391:9
[error]     at FSReqCallback.readFileAfterClose [as oncomplete] (internal/fs/read_file_context.js:63:3) {
[error]   code: 'ERR_INVALID_ARG_TYPE'
[error] }
[error] Total time: 21 s, completed Sep 21, 2020 4:01:27 PM
```

</details>

**Comment by ignasi35** [comment]
Maybe the workaround is _as simple as_  using `nvm` and require using a particular version of `node`.
cc @marcospereira

**Comment by raboof** [comment]
(I can confirm this fails on nodejs 0.14 and succeeds on 0.12)

**Comment by ignasi35** [comment]
solved in #231
